src/store/modules/password.js

```diff
@@ -0,0 +1,96 @@
+import { toast } from 'react-toastify';
+
+import { http } from '../../api/client';
+
+export const FORGOT_PASSWORD = 'FORGOT_PASSWORD';
+export const INSTRUCTIONS_SENT = 'INSTRUCTIONS_SENT';
+export const REQUEST_UNSUCCESSFUL = 'REQUEST_UNSUCCESSFUL';
+export const RESET_PASSWORD = 'RESET_PASSWORD';
+
+export const forgotPassword = () => {
+  return {
+    type: FORGOT_PASSWORD,
```

Comment:
Let's be consistent in naming actions, we append the action type to the action name. For example, if the action name is `FORGOT_PASSWORD`, we can append `SUCCESS`, `FAILED` or `UNSUCCESSFUL` to show what the action does.